flood_adapt/integrator/fiat_adapter.py

```diff
@@ -36,18 +36,8 @@ def __init__(self, model_root: str, database_path: str) -> None:
         self.bfe_path = Path(database_path) / "static" / "bfe" / "bfe.geojson"
         self.bfe_name = "bfe"
 
-    def set_hazard(self, hazard: Hazard):
+    def set_hazard(self, hazard: Hazard) -> None:
```

Comment:
see my other comment. The set_hazard functionality should be moved here

Reply:
Check